src/NServiceBus.Core/Serializers/XML/ReadonlyStream.cs

```diff
@@ -0,0 +1,47 @@
+﻿// unset
+
+namespace NServiceBus
+{
+    using System;
+    using System.IO;
+
+    class ReadonlyStream : Stream
+    {
+        ReadOnlyMemory<byte> memory;
+        long position;
+
+        public ReadonlyStream(ReadOnlyMemory<byte> memory)
```

Comment:
ReadOnly spelled same as ReadOnlyMemory
```suggestion
    class ReadOnlyStream : Stream
    {
        ReadOnlyMemory<byte> memory;
        long position;

        public ReadOnlyStream(ReadOnlyMemory<byte> memory)
```

Reply:
renamed